cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraCollectionReader.scala

```diff
@@ -58,18 +63,59 @@ object CassandraCollectionReader {
     else
       queryKeyBounds.flatMap(decomposeBounds)
 
-    val query = QueryBuilder.select("value")
-      .from(keyspace, table)
-      .where(eqs("key", QueryBuilder.bindMarker()))
-      .and(eqs("name", layerId.name))
-      .and(eqs("zoom", layerId.zoom))
-      .toString
+    val query = instance.cassandraConfig.partitionStrategy match {
+      case conf.`Read-Optimized-Partitioner` =>
```

Comment:
`ReadOptimizedPartitioner` and `WriteOptimizedPartitioner` looks better to me

Reply:
Yes - and in uncommitted work I've moved away from the idea of naming it "Partitioner" to begin with since that conflates a different idea in Cassandra.  We're not setting a partitioner... we're adjusting the strategy behind the schema optimization, so I've change the name of this around a bit.  

I'll remove the dashes though as you suggest :-)